dockstore-webservice/src/main/java/io/dockstore/webservice/core/Version.java

```diff
@@ -603,6 +614,14 @@ public void setDescriptorTypeVersionsFromSourceFiles(Set<SourceFile> sourceFiles
         getVersionMetadata().setDescriptorTypeVersions(languageVersions);
     }
 
+    public Map<Partner, Metrics> getMetricsByPlatform() {
+        return metricsByPlatform;
+    }
+
+    public void setMetricsByPlatform(Map<Partner, Metrics> metricsByPlatform) {
+        this.metricsByPlatform = metricsByPlatform;
+    }
+
```

Comment:
Will this work ok if the version is frozen?

Reply:
Agreed that this should work; it's how the version metadata works (although that's a 1-to-1).